easybuild/easyconfigs/q/Q6/Q6-20180205-gompi-2019a.eb

```
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'Q6'
+version = '20180205'
+local_commit = '5052ebe'
+
+homepage = 'https://github.com/qusers/Q6'
+description = """EVB, FEP and LIE simulator."""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+github_account = 'qusers'
+source_urls = [GITHUB_SOURCE]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['3743deac3c4f1251037b3128bdd0c850d23df2e219a7fddc5c801c82f07fefac']
+
+start_dir = 'src'
+
+buildopts = 'mpi COMP=gcc'
```

@omula Please use `$CC` rather than hardcoding `gcc` (which is important w.r.t. using `--try-toolchain` for example)

```suggestion
buildopts = 'mpi COMP="$CC"'

---

@boegel I don't think this will work because for the intel compilers it checks if the variable is ifort.
https://github.com/qusers/Q6/blob/5052ebe188f8a10a1a96094b61a7386f0e694bb2/src/makefile#L41
For gfortran it checks if the value of COMP var is gcc.
https://github.com/qusers/Q6/blob/5052ebe188f8a10a1a96094b61a7386f0e694bb2/src/makefile#L118

---

🤦‍♂ 

OK, fine. :)